eng/scripts/Install-LinuxArm64CrossCompileToolchain.ps1

```diff
@@ -0,0 +1,56 @@
+#!/bin/env pwsh
+#Requires -Version 7
+
+[CmdletBinding()]
+param()
+
+$ErrorActionPreference = 'Stop'
+
+$runtime = [System.Runtime.InteropServices.RuntimeInformation]::RuntimeIdentifier
+$parts = $runtime.Split('-')
+$os = $parts[0]
+$arch = $parts[1]
+
+if ($os -ne 'linux') {
+    Write-Host "Skipping arm64 cross-compilation toolchain installation on non-Linux host (runtime: $runtime)" -ForegroundColor Yellow
+    return
+}
+
+if ($arch -ne 'x64') {
+    Write-Host "Skipping arm64 cross-compilation toolchain installation on non-x64 Linux host (runtime: $runtime)" -ForegroundColor Yellow
+    return
+}
+
+# The bash script below follows the official documentation from .NET AOT Team.
+# https://learn.microsoft.com/en-us/dotnet/core/deploying/native-aot/cross-compile#linux
+
+$bashScript = @"
```

Comment:
Feels weird to write a ps script just to run a bash script. Not wrong per se, just a little weird. For our repos we usually stick with just bash on Unix. Up to you, though.

Reply:
Yep, this was based on feedback from EngSys. First, we need a script file so that developers using Linux-x64 machines can also install the cross-compilation tools. Second, we expect all developers to have cross-platform PowerShell installed (as noted in [contributing.md](https://github.com/Azure/azure-mcp/blob/main/CONTRIBUTING.md#prerequisites) prerequisites) regardless of OS/arch, and since the eng/scripts directory contains PowerShell scripts for all purposes, this one is also provided as a PowerShell script for consistency.